### PR TITLE
Css generator updates

### PR DESCRIPTION
**Pull request by mark-kraemer**

### CODE REVIEW

File: lib/generators/rolemodel/css/base/templates/app/views/styleguide/index.html.slim [hunk 1]
```diff
@@ -1,202 +1,236 @@
...
+    title Style Guide
+    = csrf_meta_tags
+    = csp_meta_tag
+    = stylesheet_pack_tag 'stylesheets', 'data-turbolinks-track': 'reload'
+    meta name="viewport" content="width=device-width, initial-scale=1"
+  body class="#{'flex' if params[:sidebar]}"
+    - if params[:sidebar]
+      = render 'shared/sidebar'
+    - else
+      = render 'shared/top_nav'
```
Comment (calebwoods) [inline lib/generators/rolemodel/css/base/templates/app/views/styleguide/index.html.slim:13]: Once someone chooses a layout, we might want them to actually use that in styleguide. Is there a simple way to make that migration path easier?

File: lib/generators/rolemodel/css/base/templates/app/javascript/stylesheets/components/sidebar.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  background: var(--color-white);
+  box-shadow: var(--shadow);
+
+  a {
+    text-decoration: none;
+  }
+}
+
+.sidebar-nav__item {
+  margin-bottom: 1rem;
```
Comment (calebwoods) [inline lib/generators/rolemodel/css/base/templates/app/javascript/stylesheets/components/sidebar.scss:19]: This should use a `var`

**Comment by calebwoods** [comment]
@mark-kraemer can you rebase this so we can get this integrated? I know the Shell Builder team is looking to use this pattern

File: lib/generators/rolemodel/css/base/templates/app/javascript/stylesheets/components/sidebar.scss (hunk 2 above)
Comment (scriswell) [reply lib/generators/rolemodel/css/base/templates/app/javascript/stylesheets/components/sidebar.scss:19]: Ideally we would not use a margin here and instead use gap on a parent element.

**Comment by scriswell** [comment]
@mark-kraemer @calebwoods this is an old PR. Is it still something we want merged in?

**Comment by Jeremy-Walton** [comment]
This PR is not relevant anymore. CSS has been moved to https://github.com/RoleModel/rolemodel-design-system

**Pull request closed without merging**